README.md

```diff
@@ -82,7 +82,7 @@ spec:
 * `--http-endpoint <endpoint>`: The TCP network address where the HTTP server for diagnostics, including CSI driver health check and metrics. The default is empty string, which means the server is disabled.
 Several klog arguments such as `-v <log level>` and `-logging-format <log format>` are supported.
 
-* Some klog arguments are supported, such as `-v <log level>` or `-logging-format <log format>`.
+* [Arguments set by the `k8s.io/component-base/logs` package for klog](https://github.com/kubernetes/component-base/blob/v0.28.0-rc.0/logs/api/v1/options.go#L337-L355) are supported, such as `-v <log level>` and `-logging-format <log format>`.
```

Comment:
I believe we use `--` instead of `-`. Please update line 83 and 85.

Reply:
Sorry, I removed the mistake on line 83. I've also fixed the description on line 85 to use `--`.

Reply:
Thanks. I squashed the commits.